test/grouped_convnd_fwd/test_grouped_convnd_fwd_multi_ab_interface.cpp

```diff
@@ -194,6 +196,10 @@ class TestGroupedConvndFwdInterface
 
 TEST_F(TestGroupedConvndFwdMultiAInterface, MultiA)
 {
+    if(!ck::is_xdl_supported())
+    {
+        GTEST_SKIP();
+    }
```

Comment:
Isn't it better to put it inside`this->Run` function ?

Reply:
Moved to SetUp function.